### DISCUSSION TITLE
ToolBar里的内容怎么更新？

### DISCUSSION THREAD

**Discussion by funyaliga** [antvis/G6#3371] [Q&A]
现在需求是toolbar里有个按钮是“展开所有”，当点击后，按钮文案要变成“收起所有”。

但目前不知道怎样的update toolbar里的内容。

求帮帮，感谢🙏

**Comment by Yanyan-Wang** [comment]
是你自己加的按钮么？那可能就是通过 id 获取那个 DOM 去改吧
